src/lib/stores/wallet.ts

```
@@ -338,197 +425,286 @@ const createTransactionsStore = () => {
+                "[WalletStore] Payment pending - lockup transaction broadcast",
+              );
+              // Track pending tx for fast polling (Breez SDK uses Liquid network)
+              if (isValidTxid(event.details?.txId)) {
+                trackPendingTx(event.details.txId, "liquid");
+              } else if (event.details?.txId) {
+                logInvalidTxid(event.details.txId, event.details);
+              }
+              walletStore.refresh();
+              transactions.refresh();
+              // Only notify if initial sync is complete (prevents showing stale pending payments)
+              const currentState = get(walletStore);
```

Biome (2.3.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)